### DISCUSSION TITLE
How to see the `console` button on the bottom left of the browser preview in Codesandbox Repositories?

### DISCUSSION THREAD

**Discussion by SelfDevTV** [codesandbox/codesandbox-client#7427] [Q&A] [answered]
Hi guys,

simple question:

* When I use the normal Codesandbox (without repositories), on the browser preview I have a nice little javascript console button:
![image](https://user-images.githubusercontent.com/48333000/218691092-a6f68b6d-125d-485c-9995-1e77576582ca.png)

* When I create a new codesandbox in `Repositories` I have a web view and a terminal, but no way I can show the javascript console with this neat little button. 

So my question is, where can I find it or am I supposed to use the normal Browsers Javascript console? 

I kinda liked the built in console since it only shows logs from the app running, instead of my browser that shows so much extra stuff from extensions / codesandbox website itself etc. in the console.

I also found no clues about that in the docs, maybe that might be worth documenting. 

Thx guys,
Geralt

**Comment by JayaKrishnaNamburu** [comment, marked as answer]
@SelfDevTV thanks for reaching out. Yes at the moment we need to use browser console for the output. But we are actively working on the repositories to add new feature. And the `console` is also in the list. We will bring it soon 😄

**Reply by SelfDevTV** [reply]
Awesome thank you so much !
